doc/libpmemset/pmemset_set_contiguous_part_coalescing.3.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMSET_SET_CONTIGUOUS_PART_COALESCING, 3)
+collection: libpmemset
+header: PMDK
+date: pmemset API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmemset_set_contiguous_part_coalescing.3 -- man page for pmemset_set_contiguous_part_coalescing)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmemset_set_contiguous_part_coalescing**() - set part coalescing feature in the pmemset
+
+# SYNOPSIS #
+
+```c
+#include <libpmemset.h>
+
+struct pmemset;
+
+int pmemset_set_contiguous_part_coalescing(struct pmemset *set, bool value);
+```
+
+# DESCRIPTION #
+
+The **pmemset_set_contiguous_part_coalescing**() sets part coalescing feature flag in
+pmemset to the provided value.
+
+When part coalescing feature is turned on each part mapped, using **pmemset_part_map**(3) function,
```

Comment:
missing comma? This paragraph is hard to read :)

`When part coalescing is enabled, the pmemset_part_map function will ... `
?

Generally, when writing documentation, try to use active voice and use the simplest possible word variant.